tls/s2n_cipher_preferences.c

```diff
@@ -273,6 +273,35 @@ const struct s2n_cipher_preferences cipher_preferences_20170210 = {
     .allow_chacha20_boosting = false,
 };
 
+/*
+ * TLS1.3 support.
+ * FIPS compliant.
+ * No DHE (would require extra setup with s2n_config_add_dhparams)
+ */
+struct s2n_cipher_suite *cipher_suites_20230317[] = {
+    /* TLS1.2 with ECDSA */
+    &s2n_ecdhe_ecdsa_with_aes_128_gcm_sha256,
+    &s2n_ecdhe_ecdsa_with_aes_256_gcm_sha384,
+    &s2n_ecdhe_ecdsa_with_aes_128_cbc_sha256,
```

Comment:
aes-cbc-hmac256 is typically a lot slower than aes-cbc-hmac-sha1 but pretty much all modern clients that care about performance support GCM anyway